patches/chrome-android-javatests-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/javatests/BUILD.gn b/chrome/android/javatests/BUILD.gn
+index 09485ce2bb525b1d9276e8facd81878c547e77ac..096e640ba23c082997b89f71216a0430c968f030 100644
+--- a/chrome/android/javatests/BUILD.gn
++++ b/chrome/android/javatests/BUILD.gn
+@@ -1712,6 +1712,7 @@ android_library("chrome_unit_test_java_various") {
+     "src/org/chromium/chrome/browser/toolbar/ToolbarSecurityIconTest.java",
+     "src/org/chromium/chrome/browser/webapps/WebappLaunchCauseMetricsTest.java",
+   ]
++  sources -= [ "src/org/chromium/chrome/browser/ntp/IncognitoDescriptionViewRenderTest.java" ]
```

Comment:
why are we adding a test to this?

Reply:
I am not sure, it came from https://github.com/brave/brave-core/pull/25850#discussion_r1790916189 .
I'll put the answer here when I'll figure out.

Reply:
After discussion with @simoarpe  it turned out we don't need this patch.

It's purpose was to prevent CI build errors for https://github.com/brave/brave-core/pull/25850, the PR changed the layout which was used by the test. Right now it is hard to recall more details.

On Android we don't run Chromium tests, so the target `chrome_unit_test_java_various` and the file `IncognitoDescriptionViewRenderTest.java` must not be compiled for us. Tried without patch - there was no issues neither on `cr134` nor on `master`.

So  the patch was removed at https://github.com/brave/brave-core/pull/27172/commits/2783a10331bdcd330d689e0184f2a4cc80f6fe31

Thanks for pointing to this issue.